runtime/heiko/src/constants.rs

```diff
@@ -16,7 +16,7 @@ pub mod currency {
     use primitives::Balance;
     pub const EXISTENTIAL_DEPOSIT: u128 = 500;
 
-    pub const MILLICENTS: Balance = 1_000_000_000;
+    pub const MILLICENTS: Balance = 10_000_000;
```

Comment:
I am assuming that these currency parameters and balance storage are related. Feel free to ignore this comment if they are not.

If a millicent is 10^-3 cent and a cent is 10^-3 of an integral unit of value, then a integral unit of value is 10^-6 millicents or `0.000001 * 1000000`.
Parallel uses `FixedU128` that has 18 decimals so a milicent should be 10^12 instead of 10^7

```rust
let _ = FixedU128::from_inner(FixedU128::DIV / 1_000_000); // 0.000001
```

IIRC, `10_000_000` is related to the number of decimals used by one of the parity's runtimes

Reply:
I think `FixedU128` is not related to decimals, substrate's default decimal is 12, so I think I just did it like this, the previous one was a bit strange for me